src/js/modules/articles/components/ArticleHeader.js

```diff
@@ -94,7 +94,9 @@ const ArticleHeader = ({ classes, article }) => {
         </Link>
         <h1 className={classes.headline}>{article.title}</h1>
         <div className={classes.meta}>
-          <Byline classes={classes} contributors={article.contributors} />
+          {contributors.length > 0 &&
```

Comment:
When does this happen?

Reply:
There are no contributors on the recent big sibs Undercurrents article. @DJankauskas I'm not sure if that was a mistake or if the contributor wanted to remain anonymous.

If the latter case let's open an issue to create a `User` to represent anonymity.